travis/build-and-test.sh

```diff
@@ -28,14 +28,24 @@ pushd "$BUILD_DIR"
 
 # configure build
 if [ "$ENABLE_COVERAGE" == "On" ]; then
-    cmake "$REPO_ROOT" -DENABLE_COVERAGE=On
+    cmake "$REPO_ROOT" -DENABLE_COVERAGE=On -DCMAKE_INSTALL_LIBDIR=lib
     make -j$(nproc) coverage
 else
-    cmake "$REPO_ROOT"
+    cmake "$REPO_ROOT" -DCMAKE_INSTALL_LIBDIR=lib
```

Comment:
Why are you setting `CMAKE_INSTALL_LIBDIR` manually?

Reply:
We need to know the exact path were the cmake config files are installed.

https://github.com/AppImage/libappimage/blob/9581596dee0a95abacc954d48b44b5e37e7202a3/travis/build-and-test.sh#L49